src/gui/osutils/nixutils/NixUtils.cpp

```diff
@@ -288,8 +289,8 @@ bool NixUtils::triggerGlobalShortcut(uint keycode, uint modifiers)
 
 bool NixUtils::registerGlobalShortcut(const QString& name, Qt::Key key, Qt::KeyboardModifiers modifiers, QString* error)
 {
-#ifdef WITH_XC_X11
-    auto keycode = XKeysymToKeycode(dpy, qcharToNativeKeyCode(key));
+#ifdef WITH_X11
+    auto keycode = XKeysymToKeycode(dpy, qcharToNativeKeyCode(QChar(key)));
```

Comment:
I had to change this to `auto keycode = XKeysymToKeycode(dpy, qcharToNativeKeyCode(QChar(static_cast<uint>(key))));` to get it to build on my setup.

Reply:
I this pr already usable?

Reply:
> I this pr already usable?

It builds, runs and can open my KDBX database. I haven't tested further. There might be a showstopping bug somewhere that @varjolintu isn't happy with since he mentioned it "needs more work"... I can't tell.

Reply:
Everything just needs to be checked. It's possible that we merge this when possible and the do more fixes later when possible problems are found.

Reply:
It doesn't make sense to force a `Qt::Key` to `QChar`. We should use `qtToNativeKeyCode` directly.

```suggestion
    auto keycode = XKeysymToKeycode(dpy, qtToNativeKeyCode(key));
```

Reply:
@Aetf This is on my fix list too. I'll update this branch this week when I have the time.